src/hotspot/share/oops/arrayStorageProperties.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
(Comment for the whole file)
I understand this is the first stage of a set of complex changes to remove the `ArrayStorageProperties` code, but cleaning up the `ArrayStorageProperties` as things are getting removed might help to see progress (and would make reviews easier).
For instance:
  - `clear_flattened()`
  - `set_flattened()`
  - `set_null_free()`
  - `for_signature(Symbol* sig)`
seem to not be used anymore and could be removed.

Reply:
Correct, ultimately the whole thing can go...but yes, they can go now.